### PR TITLE
Clarify the Criteria

### PR DESCRIPTION
**Pull request by yuya-takeyama**
@quipper/sre 
Please review and merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 - Use at least 1 programming code. You can use any kind of programming languages.
 - Use at least 1 database which can use SQL or SQL-like query, such as PostgreSQL, MySQL, sqlite3, Google BigQuery, InfluxDB, etc...
+- Consider the real-world problems you think (e.g. performance, maintainability)
```
Comment (tominarit) [inline README.md]: Take in real-world problems into consideration (e.g. performance, maintainability)
が個人的にはしっくりきます...!

File: README.md [hunk 2]
```diff
@@ -23,6 +24,7 @@
   - The name of programming language you use
   - The name of database software you use
   - The way how to run the program you create
+  - What aspects you considered (or compromised) about
```
Comment (tominarit) [inline README.md]: What are the main aspects that you took into consideration (or compromised).

File: README.md (hunk 1 above)
Comment (drewpterry) [inline README.md]: Take into consideration problems you may face in the real-world (e.g. performance, maintainability etc)

File: README.md (hunk 2 above)
Comment (drewpterry) [inline README.md]: Aspects you considered and compromises you made when completing the task

**Comment by yuya-takeyama** [comment]
@tominarit @drewpterry 
Thank you both!
This time, I took @tominarit's one.

But I learned a lot from both ones!

@quipper/sre 
Now it's ready.

**Comment by lamanotrama** [comment]
👍

**Pull request merged**